ui.xml

```diff
@@ -180,6 +180,36 @@
         </VerticalLayout>
     </VerticalLayout>
 </Panel>
+<Panel id="panelFear" visibility="Invisible" allowDragging="true" returnToOriginalPositionWhenReleased="false" width="170" height="80" offsetXY="315 0" rectAlignment="UpperLeft">
```

Comment:
can you add buttons onto UI for  +/- fear?

Reply:
As I mentioned, I'd be happy to do this, but would prefer to leave it for a followup.